src/Components/sharedComponents/BetaPath.ts

```diff
@@ -0,0 +1,6 @@
+import { useGetEnvironment } from '../../Utilities/useGetEnvironment';
+
+export const BetaPath = (path: string) => {
+  const { isBeta } = useGetEnvironment();
+  return isBeta() ? `/preview/${path}` : path;
+};
```

Comment:
This definitelly works, but I was imagining something like:
```suggestion
export const consolePath = (path: string, beta: boolean) => {
  return beta ? `/preview/${path}` : path;
};
```

It would ease up the worries about using custom hook conditionally somewhere. We'd evaluate isBeta in the caller: `consolePath(path, isBeta())`.

Up to you, your solution also works ;)

Reply:
Aah, right! Yeah I've moved the hook call outside of the helper function. I agree treating it as a component was quite awkward :sweat_smile: